templates/cbt/V/templates.yaml

```diff
@@ -0,0 +1,43 @@
+dataset: cbt
+subset: V
+templates:
+  0725fe5e-1bba-4e08-a448-9e0038164914: !Template
+    id: 0725fe5e-1bba-4e08-a448-9e0038164914
+    jinja: '{{sentences | join('''')}}
+
+
+      Write the next sentence of this story.
+
+      |||
+
+      {{ question.replace("XXXXX", answer) }}'
+    name: next_sentence_generation
+    reference: Generate the next sentence given the story.
+  2c326181-dbba-401e-accb-d84ea0162f0a: !Template
+    id: 2c326181-dbba-401e-accb-d84ea0162f0a
+    jinja: "{{ sentences | join('') }}
+
+      {{question}}
+
+      Which part-of-speech fill replaces XXXXX the best?
+
+      |||
+
+      Verb"
+    name: pos_prediction
+    reference: Determine which part of speech will best fit in the blank.
+  b26cae56-1fbd-47a5-8c8d-d981ca098239: !Template
+    id: b26cae56-1fbd-47a5-8c8d-d981ca098239
+    jinja: '{{sentences | join ('''')}} {{question}}
+
+
+      Which of the following options replaces XXXXX the best?
```

Comment:
Can you put the task description first? It will help when truncating to not truncate the task description.
Will need some marginal wording reformulation

Reply:
same for pos_prediction